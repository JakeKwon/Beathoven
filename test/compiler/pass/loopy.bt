int x = 0;

while (x < 3) {
  if (x == 2){
    print("y\n");
  }
  else {
    print("x\n");
  }
  x=x+1;
}