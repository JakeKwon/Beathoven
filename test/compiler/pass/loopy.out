x
x
y
7
6
5